pkg/sql/logictest/testdata/logic_test/alter_column_type

```diff
@@ -161,3 +161,197 @@ ALTER TABLE t ALTER s TYPE STRING
 
 statement ok
 DROP TABLE t
+
+subtest alter_column_type_general
+
+statement ok
```

Comment:
State above each test what you are trying test -- it makes the tests easier to follow.